cmake-toolchain/valgrind-suppressions.txt

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
How is this file generated?

Reply:
Using the `--gen-suppressions` argument to valgrind.  Use`--suppressions` to pass the output from `--gen-suppressions` back into valgrind. You can customize the name so that it is clear what is being suppressed and I did that here with `SensorTest.ScanEomOneMatch.clangpp`.  More details about suppressions in the valgrind manual https://valgrind.org/docs/manual/manual-core.html#manual-core.suppress